client/types/objectsBlockType.ts

```diff
@@ -0,0 +1,11 @@
+import type { ArgumentObject } from './argumentObjectsType';
```

Comment:
Maybe we could have these two types in the same file, since they seem to work together ?